lib/measures/envelope_and_internal_load_breakdown/resources/report.html.erb

```diff
@@ -37,6 +37,7 @@
 
 
 
+
```

Comment:
are these the random additions that are happening?

Reply:
Well, not random happens for a few measures every time update measures rake task is called, but  `openstudio measure -u` in the CLI.

Reply:
Can figure out why it happens here and bypass it up resolve it in extension gem. Then can go back and clean up blank lines in the measures.